packages/e2e-tests/testFixtureCloneRecording.ts

```diff
@@ -31,34 +31,39 @@ const testWithCloneRecording = base.extend<TestIsolatedRecordingFixture>({
     const testScope = crypto.randomUUID();
     let newRecordingId: string | undefined = undefined;
     try {
-      const { recording } = exampleRecordings[exampleKey];
-      console.log("Cloning recording");
-      newRecordingId = await cloneTestRecording(recording);
+      try {
+        const { recording } = exampleRecordings[exampleKey];
+        console.log("Cloning recording");
+        newRecordingId = await cloneTestRecording(recording);
+      } catch (err: any) {
+        if (axios.isAxiosError(err)) {
+          console.error("Axios error cloning recording: ", {
+            errors: err.response?.data?.errors,
+            details: err.toJSON(),
+          });
+        } else {
+          console.error("Error cloning recording: ", err);
+        }
+        throw err;
+      }
 
       try {
         await loadRecording(newRecordingId);
       } catch (e) {
         console.warn("Error processing recording; ignoring.");
       }
 
+      // Start coverage.
       await page.coverage.startJSCoverage({
         resetOnNavigation: false,
       });
+
+      // Run test.
       await use({
         page,
         recordingId: newRecordingId,
         testScope,
       });
-    } catch (err: any) {
```

Comment:
This is the error handling for cloning, but it accidentally wrapped the entire test.